third-party/realdds/include/realdds/topics/image/image-msg.h

```diff
@@ -34,14 +43,27 @@ class image
 public:
     using type = raw::device::imagePubSubType;
 
-    image( const raw::device::image & );
+
+    image & operator=( raw::device::image && );
```

Comment:
Someone might choose to write:
`image i( raw_image );`
Which is entirely valid, but now won't work.
Furthermore, we now have (as there are no constructors) a copy-ctor that will work, which we do not want.

We want two things:
* Enable the raw::image move constructor. This will automatically enable an `operator=` for raw::image, because it will create a temporary image object (which will be optimized away).
* Delete the default copy constructor and `operator=`. Doing this may (you have to try) require you to define the other constructors and assignments.